src/compositemark/errorbar.ts

```diff
@@ -556,7 +559,7 @@ function errorBarAggregationAndCalculation<
     for (const postAggregateCalculate of postAggregateCalculates) {
       tooltipSummary.push({
         fieldPrefix: postAggregateCalculate.as.substring(0, 6),
-        titlePrefix: replaceAll(replaceAll(postAggregateCalculate.calculate, 'datum["', ''), '"]', '')
+        titlePrefix: replaceAll(replaceAll(postAggregateCalculate.calculate, "datum['", ''), "']", '')
```

Comment:
do we use accessWithDatumToUnescapedPath here?

Reply:
Yes. 

1) The existing logic peel off `datum["`  and `"]` from the `calculate` to create a prefix. 
E.g., `datum["yh"]`  -> `yh`. 

2) I replaced the double quote datum expression with `accessWithDatumToUnescapedPath`  using the single quote.

3) To keep the existing logic, I changed the code to peel off `datum['`  and `']`